src/Quasar/QuasarF.purs

```diff
@@ -124,7 +141,7 @@ readFile
   ∷ PrecisionMode
   → FilePath
   → Maybe Pagination
-  → QuasarFE JArray
+  → QuasarFE (ExpiredContent JArray)
```

Comment:
in /Quasar/Advanced/QuasarAF.purs type of `readFile` was not changed and `Detail`-ed version was added.
Same here for `readFileEJson` and `invokeFileEJson `, but here we are changing type of `writeQuery`, why?

Reply:
Not sure if the changes to `ReadFile` are correct, I added a `ReadfileCache` but that feels wrong